packages/web-components/src/dropdown/dropdown.options.ts

```diff
@@ -0,0 +1,34 @@
+import { ValuesOf } from '@microsoft/fast-foundation';
+
+/**
+ * DropdownAppearance - dropdown appearance defined by css.
+ * @public
+ */
+export const DropdownAppearance = {
+  outline: 'outline',
+  underline: 'underline',
+  subtle: 'filled-darker',
```

Comment:
Do these have the same meaning in dark theme?

Reply:
Yes, these are all appearances that are used in both dark and light mode.

![image](https://github.com/microsoft/fluentui/assets/11062709/67d645a9-3e9d-4bf2-9087-2c013615f5ad)